login-workflow/src/new-architecture/screens/CreatePasswordScreen/CreatePasswordScreen.tsx

```diff
@@ -87,7 +87,7 @@ export const CreatePasswordScreen: React.FC<CreatePasswordScreenProps> = (props)
         showPrevious: true,
         previousLabel: t('bluiCommon:ACTIONS.BACK'),
         canGoPrevious: true,
-        currentStep: 2,
+        currentStep: 3,
```

Comment:
I'm sure this is wrong in other screens too, this is just the first time I'm looking closely at it. These values (current and total screens) should _not_ be hardcoded...they should be pulled from the RegistrationWorkflow component using the hook (if the user re-orders screens, these hardcoded values will break).

Reply:
Done.